packages/react-next/src/components/SwatchColorPicker/SwatchColorPicker.types.ts

```diff
@@ -45,24 +46,31 @@ export interface ISwatchColorPickerProps {
    */
   colorCells: IColorCellProps[];
 
+  /**
+* @deprecated Provide `selectedId` if controlled or `defaultSelectedId` if uncontrolled
```

Comment:
Some spaces got lost
```suggestion
   * @deprecated Provide `selectedId` if controlled or `defaultSelectedId` if uncontrolled
```